### ISSUE TITLE
Transfer learning

### ISSUE THREAD

**Issue by mbellitti** [mbellitti/wikiart-classifier#9]
What's the status on transfer learning? Was someone looking into it?

**Comment by mbellitti** [comment]
This looks like a good starting point, if not all there is to it: https://keras.io/applications/

**Comment by mohitpandey92** [comment]
I have implemented transfer learning using [this](https://github.com/aditya9898/transfer-learning/blob/master/transfer-learning.ipynb) link. So far, training accuracy is above 80 percentage and validation accuracy is about 40 percentage with small number of images (1000) and epochs (10).

It's on the branch "transfer_learning"
